vtkext/private/module/vtkF3DConfigure.h.in

```diff
@@ -4,6 +4,8 @@
 #include <string>
 
 // TODO: Use CMake definitions and get rid of these
+#cmakedefine01 F3D_WINDOWS_GUI
```

Comment:
Use cmake define instead ? :)

Reply:
it's actually not needed in the C++, I'll just remove it